<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCtagsSenseBrowser</class>
 <widget class="QFrame" name="qCtagsSenseBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>440</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lSearch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leSearch"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lLoading">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QStackedWidget" name="swPages">
     <widget class="QTreeView" name="tvMembers">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTreeView" name="tvSearchResult">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="pbIndexing">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>Indexing at %p%</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/qCtagsSense.qrc">
     <normaloff>:/icons/class.png</normaloff>:/icons/class.png</iconset>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
   <property name="toolTip">
    <string>Switch to Browser</string>
   </property>
  </action>
  <action name="aSearchResults">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../monkey/src/resources/resources.qrc">
     <normaloff>:/edit/icons/edit/replace.png</normaloff>:/edit/icons/edit/replace.png</iconset>
   </property>
   <property name="text">
    <string>Search Results</string>
   </property>
   <property name="toolTip">
    <string>Switch to Search Results</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/qCtagsSense.qrc"/>
  <include location="../../monkey/src/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
